url-updater/bin/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file was generated by Eclipse and should not be comitted.
You can add `bin` to `.gitignore` to avoid this in the future.